docs/ui-notes.md

```diff
@@ -0,0 +1,40 @@
+# UI Notes
+
+### Miscellaneous
+- toolbar background: `<Sheet variant='soft'>` or `<Sheet className='TapSearch__section' variant='outline' sx={{flexDirection: 'column', flexGrow: 1}}>`
+
```

Comment:
Not sure about 'TapSearch__section'.  I feel that should be replaced with Sheet or Card.

Reply:
I agree - I also didn't like that classname is being used. Again feel free to edit it.